README.md

```diff
@@ -25,119 +25,135 @@ Features:
-    - [Certificate verification](#certificate-verification)
-      - [Soft Confirmations](#soft-confirmations)
-    - [In-Memory Backend](#in-memory-backend)
-    - [Asynchronous Secondary Insertions](#asynchronous-secondary-insertions)
-    - [Storage Fallback](#storage-fallback)
-    - [Storage Caching](#storage-caching)
-    - [Failover Signals](#failover-signals)
-  - [Blob Lifecycle](#blob-lifecycle)
-    - [Posting Payloads](#posting-payloads)
-    - [Retrieving Payloads](#retrieving-payloads)
-    - [Rollup Commitment Schemas](#rollup-commitment-schemas)
-      - [Optimism Commitment Mode](#optimism-commitment-mode)
-      - [Standard Commitment Mode](#standard-commitment-mode)
+- [Overview](#overview)
+- [User Guide](#user-guide)
+  - [Quick Start With Memstore Backend](#quick-start-with-memstore-backend)
+  - [REST API Routes](#rest-api-routes)
+    - [Standard Routes](#standard-routes)
+    - [Optimism Routes](#optimism-routes)
+  - [Deployment Against Real EigenDA Network](#deployment-against-real-eigenda-network)
+  - [Features and Configuration Options (flags/env vars)](#features-and-configuration-options-flagsenv-vars)
+  - [Requirements / Dependencies](#requirements--dependencies)
+    - [Authn/Authz/Payments](#authnauthzpayments)
+    - [Ethereum Node](#ethereum-node)
+    - [SRS Points](#srs-points)
+    - [Hardware Recommendation](#hardware-recommendation)
+  - [Monitoring / Observability](#monitoring--observability)
+- [Blob Lifecycle](#blob-lifecycle)
+  - [Posting Payloads](#posting-payloads)
+  - [Retrieving Payloads](#retrieving-payloads)
+  - [Rollup Commitment Schemas](#rollup-commitment-schemas)
+    - [Optimism Commitment Mode](#optimism-commitment-mode)
+    - [Standard Commitment Mode](#standard-commitment-mode)
+- [Contributor Guide](#contributor-guide)
   - [Testing](#testing)
     - [Unit](#unit)
     - [Integration / E2E against local deployment](#integration--e2e-against-local-deployment)
     - [E2E against Holesky](#e2e-against-holesky)
     - [E2E Fuzz](#e2e-fuzz)
-  - [Metrics](#metrics)
-  - [Flags and Env Vars](#flags-and-env-vars)
-  - [Resources](#resources)
 
 
-## Deployment Guide
+## User Guide
 
-### Hardware Requirements
+### Quick Start With Memstore Backend
 
-The following specs are recommended for running on a single production server:
+For testing purposes, proxy provides a fully in-memory backend that mocks a real backing EigenDA network. Here's how to start the proxy in this mode and interact with it:
 
-* 4 GB RAM
-* 1-2 cores CPU
+```bash
+# Start the proxy with memstore backend enabled
+$ docker run --rm -p 3100:3100 ghcr.io/layr-labs/eigenda-proxy:latest --memstore.enabled --port 3100
 
-### Ethereum Node Requirements
+# In another terminal... submit a payload save the returned cert in hex format
+$ CERT_HEX=$(curl -X POST -d my-eigenda-payload "http://127.0.0.1:3100/put?commitment_mode=standard" | xxd -p | tr -d ' \n')
```

Comment:
recommend a golang example in addition that does the exact same thing

Reply:
yep will do in a later PR because I don't think golang stuff should live in README. Added https://github.com/Layr-Labs/eigenda-proxy/issues/332